modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelperTest.java

```diff
@@ -240,6 +240,56 @@ public void testBelongsToCondition() throws Exception {
 		Assert.assertTrue((boolean)ddmFormFieldPropertyChanges.get("readOnly"));
 	}
 
+	@Test
+	public void testInvalidConfirmationValueWithTextField() throws Exception {
```

Comment:
Hi @natocesarrego, these tests already cover both fields. 
Since the evaluation only checks whether `requireConfirmation` is true, the field type makes no difference in this unit test.